modules/running-network-verification-manually-cli.adoc

```diff
@@ -19,56 +19,63 @@ When you run the network verification, you can specify a set of VPC subnet IDs o
 ----
-+
-[TIP]
-====
-To output the full list of verification tests, you can include the `--debug` argument when you run the `rosa verify network` command.
-====
-+
-** Verify the network configuration by specifying the VPC subnets IDs:
+*** Ensure that verification to all subnets has been completed.
 +
 [source,terminal]
 ----
-$ rosa verify network --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
+$ rosa verify network --watch \ <1>
+                      --status-only \ <2>
+                      --region <region_name> \ <3>
+                      --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc <4>
 ----
+<1> The `watch` flag causes the command to complete after all the subnets under test are in a failed or passed state.
+<2> The `status-only` flag does not trigger a run of network verification but returns the current state, for example, `subnet-123 (verification still in-progress)`. By default, without this option, a call to this command always triggers a verification of the specified subnets.
+<3> Use a specific AWS region that overrides the _AWS_REGION_ environment variable.
+<4> Enter a list of subnet IDs separated by commas to verify. If any of the subnets do not exist, the error message `Network verification for subnet 'subnet-<subnet_number> not found` displays and no subnets are checked.
 +
 .Example output
 [source,terminal]
 ----
-E: Validating Subnet subnet-03146b9b52b6024cb egress
-E: X Egress failed to https://events.pagerduty.com
+I: Checking the status of the following subnet IDs: [subnet-03146b9b52b6024cb subnet-03146b9b52b2034cc]
+I: subnet-03146b9b52b6024cb: passed
+I: subnet-03146b9b52b2034cc: passed
 ----
 +
-** Verify the network configuration by specifying the VPC subnets IDs and a proxy URL:
+[TIP]
+====
+To output the full list of verification tests, you can include the `--debug` argument when you run the `rosa verify network` command.
+====
++
+** Verify the network configuration by specifying the VPC subnets IDs, replacing `<region_name>` with your AWS region:
 +
 [source,terminal]
 ----
-$ rosa verify network --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc \
-                      --additional-trust-bundle-file /path/to/ca.cert \
-                      --https-proxy <proxy_url> <1>
+$ rosa verify network --status-only --region <region_name> --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
 ----
-<1> Replace `<proxy_url>` with the URL of your proxy service, for example `\https://10.10.0.1`.
 +
 .Example output
 [source,terminal]
 ----
-I: Using proxy configuration
-I: Subnet IDs detected:  subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
-
-E: Validating Subnet subnet-03146b9b52b6024cb egress
-E: X Egress failed to https://events.pagerduty.com
+$ rosa verify network --region us-east-1 --status-only --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc
```

Comment:
The command is not correct.
1. The command should not contain "status-only" as the flag does not trigger a run of network verification
2. Check  network verification via subnets command should like this : rosa verify network \
      --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc  \
      --role-arn arn:aws:iam::765374464689:role/my-Installer-Role


rosa verify network --region us-east-1 --status-only --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc

Reply:
> The command is not correct.
> 
> 1. The command should not contain "status-only" as the flag does not trigger a run of network verification
> 2. Check  network verification via subnets command should like this : rosa verify network 
>    --subnet-ids subnet-0b761d44d3d9a4663,subnet-0f87f640e56934cbc  
>    --role-arn arn:aws:iam::765374464689:role/my-Installer-Role
> 
> rosa verify network --region us-east-1 --status-only --subnet-ids subnet-03146b9b52b6024cb,subnet-03146b9b52b2034cc

Thanks @yingzhanredhat . I have amended this command.

Reply:
@yingzhanredhat When I try to run this command, I get an error message. It seems the region is required: 
```
$ rosa verify network --subnet-ids subnet-07e040767f6a7dcaa,subnet-03090605cc9803257 --role-arn arn:aws:iam::269733383066:role/ManagedOpenShift-Installer-Role
E: Region is required

``` 
The command is successful with the `--region` flag: 
```
$ rosa verify network --subnet-ids subnet-07e040767f6a7dcaa,subnet-03090605cc9803257 --region us-east-1 --role-arn arn:aws:iam::269733383066:role/ManagedOpenShift-Installer-Role
I: Verifying the following subnet IDs are configured correctly: [subnet-07e040767f6a7dcaa subnet-03090605cc9803257]
I: subnet-07e040767f6a7dcaa: passed
I: subnet-03090605cc9803257: pending
I: Run the following command to wait for verification to all subnets to complete:
rosa verify network --watch --status-only --subnet-ids subnet-07e040767f6a7dcaa,subnet-03090605cc9803257

```

Reply:
Hi @thomasmckay. Can you please advise on whether `--region` should be used as part of this command? Thanks.

Reply:
yes, `--region` is required

Reply:
Thanks @thomasmckay.